mod/groups/views/default/resources/groups/members.php

```diff
@@ -53,11 +64,12 @@
 
 $content = elgg_list_relationships($options);
 
-$params = [
+// build page
+$body = elgg_view_layout('content', [
```

Comment:
could be default

Reply:
was already a `content` layout, not changing it now